docs/reference/ml/apis/dfanalyticsresources.asciidoc

```diff
@@ -0,0 +1,49 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ml-dfanalytics-resources]]
+=== {dfanalytics-cap} resources
+
+A {dfanalytics} configuration object has the following properties:
+
+`analysis`::
+  (object) The type of analysis that is performed on the `source`. For example: 
```

Comment:
Perhaps it's worth adding a link to the analysis configuration objects section below?